jupyter-notebooks/analytics/user-guide/03_change_detection.ipynb

```diff
@@ -527,6 +528,16 @@
     "    # Select only the items with selected observed date\n",
     "    month_items = items_gdf.loc[items_gdf['observed'] == observed]\n",
```

Comment:
I tried to reproduce your original problem by adding this line
```
month_items = month_items[:1]
```
and trying the map rendering with and without your new code. The "Dynamic Map" that your current code changes works fine with month_items length of 1. The heatmap, however, crashes. See my other comment about https://github.com/planetlabs/notebooks/pull/155/files#r497755572

Reply:
I think under the covers the test subscription we were using that originally only had one entry now looks like it has two detections; I'll move the code as indicated.